src/Pim/Bundle/EnrichBundle/Resources/public/js/form/common/column.js

```diff
@@ -37,6 +55,43 @@ define(
              */
             _toggleColumn: function (e) {
                 $(this.$el).toggleClass('AknColumn--collapsed');
+                this._setCollapsed($(this.$el).hasClass('AknColumn--collapsed'));
+            },
+
+            /**
+             * Returns true if the column is collapsed.
+             * It uses the session storage with a key attached to this module.
+             * If no key was found, returns false by default.
+             *
+             * @returns {boolean}
+             */
+            _isCollapsed: function () {
+                var result = sessionStorage.getItem(this._getSessionStorageKey());
+                if (null === result) {
+                    this._setCollapsed(false);
```

Comment:
You're in a getter, don't change the data. It looks like you need a default value to be set elsewhere, no?